etc/config/c++.amazon.properties

```diff
@@ -322,7 +322,7 @@ compiler.clang_lifetime.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++fil
 compiler.clang_lifetime.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.clang_lifetime.semver=(experimental -Wlifetime)
 compiler.clang_lifetime.options=--gcc-toolchain=/opt/compiler-explorer/gcc-8.2.0 -Wlifetime
-compiler.clang_lifetime.notification=Lifetime profile checker based on Herb Sutter's paper; see <a href="https://herbsutter.com/2018/09/20/lifetime-profile-v1-0-posted/" target="_blank" rel="noopener noreferrer">this blog post <sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a> for more information
+compiler.clang_lifetime.notification=Lifetime profile checker based on Herb Sutters paper; see <a href="https://herbsutter.com/2018/09/20/lifetime-profile-v1-0-posted/" target="_blank" rel="noopener noreferrer">this blog post <sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a> for more information
```

Comment:
Wait, wasn't this correct? "based on Herb Sutter [his] paper", right? https://grammar.yourdictionary.com/punctuation/apostrophe-rules.html 
The apostrophe could also be behind the name, but it should be there afaik?

Reply:
My mistake. I shouldn't touch this file at all.

(I removed it 'temporary' because it boke highlighing in editor.) Reverted.